app/src/main/java/com/randomappsinc/studentpicker/editing/EditNameListFragment.java

```diff
@@ -159,21 +159,22 @@ void addNameWithVoice() {
 
     @Override
     public void onRenameChosen(String name) {
-        ListInfo listInfo = namesAdapter.getListInfo();
-        int currentAmount = listInfo.getInstancesOfName(name);
-        renameDialog.startRenamingProcess(name, currentAmount);
+        renameDialog.startRenamingProcess(name, amountOfEditChoiceFlow(name));
     }
 
     @Override
     public void onDeleteChosen(String name) {
-        ListInfo listInfo = namesAdapter.getListInfo();
-        int currentAmount = listInfo.getInstancesOfName(name);
-        deleteNameDialog.startDeletionProcess(name, currentAmount);
+        deleteNameDialog.startDeletionProcess(name, amountOfEditChoiceFlow(name));
     }
 
     @Override
     public void onDuplicationChosen(String name) {
```

Comment:
We should try to keep the user experience and the implementation as harmonized as possible so our code is easy to read and navigate. Let's change all the duplication terminology. So we should change this to "onNameAdjustmentChosen".